hardware/Dummy.cpp

```diff
@@ -75,8 +75,10 @@ namespace http {
 			if ((idx == "") || (ssensortype.empty()) || (ssensorname.empty()))
 				return;
 
-			bool bCreated = false;
-			int iSensorType = atoi(ssensortype.c_str());
+			unsigned int fullType = std::stoul(ssensortype, nullptr, 16);
```

Comment:
i think you can use atoi and cast this to a uint16_t ?
uint16_t var = (uint16_t)atoi(ssensortype.c_str());

Reply:
No problem

Reply:
Sorry yes you are right, so a standard std::stringstream could work

uint16_t iSensorType;
std::stringstream si;
si << std::hex << ssensortype;
si >> iSensorType;

Reply:
Done